argument.go

```diff
@@ -27,6 +27,7 @@ type Arg interface {
 	GetOpts() *Options
 	GetSname() string
 	GetLname() string
+	IsFlag() bool
```

Comment:
This IsFlag line is unrelated to the topic of the pull request, but it was useful for me to get a nice looking help menu. Example:
```
Options:
  -h  --help      Print help information.
  -V  --version   Concards build information..
  -r  --review    Show cards available to be reviewed.
  -m  --memorize  Show cards available to be memorized.
  -d  --done      Show cards not available to be reviewed or memorized.
  -p  --print     Prints all cards, one line per card.
  -n  --number n  How many cards to review.
  -E  --editor E  Which editor to use. Defaults to "$EDITOR".
  -M  --meta M    Path to meta file. Defaults to "$CONCARDS_META" or "~/.concards-meta".
```

Note that in my custom help menu function, if the argument is a flag, I'm not printing an argument name. And conversely, if the argument is not a flag, there is an argument name (same as the short name).

Reply:
That should be its own PR please. 1 PR == 1 functionality change.

Also that is not really common way for building help message. Arguably there are no standards of doing this per se. But there are recognized common formats for help messages and that wouldn't be it.